cmd/builder/internal/command.go

```diff
@@ -90,7 +90,7 @@ func initConfig() error {
 
 	// load the config file
 	if err := k.Load(file.Provider(cfgFile), yaml.Parser()); err != nil {
-		cfg.Logger.Error("failed to load config file", zap.String("config-file", cfgFile), zap.Error(err))
+		cfg.Logger.Fatal("failed to load config file", zap.String("config-file", cfgFile), zap.Error(err))
```

Comment:
What about 
```suggestion
		return fmt.Errorf("failed to load config file %q: %w", cfgFile, err)
```

and let the caller handle the error?

or even 

```suggestion
		cfg.Logger.Error("failed to load config file", zap.String("config-file", cfgFile), zap.Error(err))
		return fmt.Errorf("failed to load config file %q: %w", cfgFile, err)
```

which is what we do below for `UnmarshalWithConf` errors.